yarn.lock

```diff
@@ -9,18 +9,38 @@
   dependencies:
     "@babel/highlight" "^7.0.0"
 
+"@babel/core@7.1.0":
```

Comment:
There are quite a lot of changes inside `yarn.lock`. Are you able to run it on `Android`?
Because for me the lockfile related problem that we solved few days ago is back.

<img width="374" alt="screenshot 2019-01-09 at 15 47 09" src="https://user-images.githubusercontent.com/2660330/50907046-bb961a00-1426-11e9-9a9a-de7353b23d05.png">

So I'm just thinking if it's not the same case.

Reply:
It was working before, but I've rebased it with master later. I'm going to resolve conflicts and rebase it again.

Reply:
Ok, thanks.